File: NET/kamifer.LogManager/Destinations/ConsoleDestiny.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace kamifer.LogManager.Destinations
{
    public class ConsoleDestiny : Destiny
    {

        public override void sendLog(string texto, string Level)
        {
            String ver = getReformatedText(texto, Level);
		    Console.WriteLine(ver);
        }

         

        public override void trace(String texto)
        {
            this.sendLog(texto, Logger.LoggerLevels.TRACE);
        }


        public override void debug(String texto)
        {
            this.sendLog(texto, Logger.LoggerLevels.DEBUG);
        }


        public override void info(String texto)
        {
            this.sendLog(texto, Logger.LoggerLevels.INFO);
        }


        public override void warning(String texto)
        {
            this.sendLog(texto, Logger.LoggerLevels.WARN);
        }


        public override void error(String texto)
        {
            this.sendLog(texto, Logger.LoggerLevels.ERROR);
        }


        public override void fatal(String texto)
        {
            this.sendLog(texto, Logger.LoggerLevels.FATAL);
        }
                
    }
}

File: JAVA/src/kamifer/logManager/ConsoleDestiny.java
package kamifer.logManager;

public class ConsoleDestiny extends Destiny {



	public ConsoleDestiny() {
		
	}

	@Override
	protected void sendLog(String texto, String Level) {
		String ver = getReformatedText(texto, Level);
		System.out.println(ver);
	}



	@Override
	public void trace(String texto) {
		this.sendLog(texto, Logger.LoggerLevels.TRACE);
	}

	@Override
	public void debug(String texto) {
		this.sendLog(texto, Logger.LoggerLevels.DEBUG);
	}

	@Override
	public void info(String texto) {
		this.sendLog(texto, Logger.LoggerLevels.INFO);
	}

	@Override
	public void warning(String texto) {
		this.sendLog(texto, Logger.LoggerLevels.WARNING);
	}

	@Override
	public void error(String texto) {
		this.sendLog(texto, Logger.LoggerLevels.ERROR);
	}

	@Override
	public void fatal(String texto) {
		this.sendLog(texto, Logger.LoggerLevels.FATAL);
	}


}
